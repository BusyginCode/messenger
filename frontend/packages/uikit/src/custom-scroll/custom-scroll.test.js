import React from 'react'
import { shallow } from 'enzyme'
import CustomScroll from './custom-scroll'

describe('uikit/CustomScroll', () => {
  it('should render correctly', () => {
    expect(shallow(<CustomScroll>123</CustomScroll>)).toMatchSnapshot()
  })
})
