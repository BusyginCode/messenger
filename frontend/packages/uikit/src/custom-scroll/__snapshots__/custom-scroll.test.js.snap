// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/CustomScroll should render correctly 1`] = `
<div
  className="general"
>
  123
</div>
`;
